exporters/holstebro_helpers.py

```diff
@@ -0,0 +1,811 @@
+
+    nodes = all_nodes[org_root_uuid]
+
+    # First create list of all children
+    # then put them in hierarchial order
+
+    for node in PreOrderIter(nodes['root']):
+        # if this is only partial export of org and this org unit is not on the exports list
+        # or is a child of an org unit in the exports list, continue
+        export_node = False
+        node_parent = node
+        while(node_parent != None and export_node is False):
+            export_node = True if node_parent.name in roots[org_root_uuid] else False
+            node_parent = node_parent.parent  # look backwards in hierarchy
+
+        if not export_all and not export_node:
+            continue
+
+        ou = mh.read_ou(node.name)
+
+        if ou['org_unit_level']['name'] == 'Afdelings-niveau':
+            continue
+
+        # find this unit's manager
+        manager = find_org_manager(mh, node)
+
+        # figure out what the parent ou should be
+        # If this is only a partial export org, then add the ou in roots directly under
+        # root ou.
+
+        # Default is root uuid
+        parent_ou_uuid = org_root_uuid
+        if export_all is True:
+            parent_ou_uuid = ou['parent']['uuid'] if ou['parent'] else org_root_uuid
+        elif export_node is True and ou['uuid'] not in roots[org_root_uuid]:
+            parent_ou_uuid = ou['parent']['uuid'] if ou['parent'] else org_root_uuid
+
+        if ou['uuid'] != org_root_uuid:
+            # Do no insert root ou. This has already been added
+            child = {
+                'id': ou['uuid'],
+                'name': ou['name'],
+                'description': '',
+                'parent_id': parent_ou_uuid,
+                'staff_department': 'N',
+                'manager_id': manager['uuid'],
+                'dataFields': [data_fields],
+                'children': [],
+                'showChildren': False
+            }
+
+            logger.info(
+                f"Exporting {ou['name']} to Org. viewer with parent {parent_ou_uuid}")
+
+            # add child to parent ous list of children
+            children_ref[parent_ou_uuid].append(child)
+
+            # add uuid for this ou to children reference dict
+            # and point to this ou's list of children for later addition
+            children_ref.update({ou['uuid']: child['children']})
+
+        # Read all employees in org unit and add them to employee export rows
+        employees = mh.read_organisation_people(
+            node.name, "engagement", False)
+        for uuid, employee in employees.items():
+            address = mh.read_user_address(uuid, username=True, cpr=False)
+
+            # is this employee the manager for the department? Then fetch the parent ou's manager
+            if uuid != manager['uuid']:
+                responsible = manager['uuid']
+            else:
+                parent_manager = find_org_manager(mh, node.parent)
+                responsible = parent_manager['uuid']
```

Comment:
Jeg foretrækker tit at omskrive `if ... else ...` konstruktioner med assignments sådan her:
```suggestion
            responsible = manager['uuid']
            if uuid == manager['uuid']:
                parent_manager = find_org_manager(mh, node.parent)
                responsible = parent_manager['uuid']
```
Det er i mit hoved nemmere at forstå, men det er en smagssag.